nhais/nhais-env-example.yaml

```diff
@@ -0,0 +1,14 @@
+#Common env variables
+NHAIS_OUTBOUND_SERVER_PORT: "80"
+NHAIS_LOG_LEVEL: NOTSET
+NHAIS_OUTBOUND_MAX_RETRIES: "100"
+NHAIS_OUTBOUND_RETRY_DELAY: "3"
+
+#User-specific env variables
+NHAIS_OUTBOUND_QUEUE_BROKERS: amqp://localhost:5672
+NHAIS_OUTBOUND_QUEUE_NAME: 'nhais_outbound'
+
+#common tests
+AWS_ACCESS_KEY_ID: test
+AWS_SECRET_ACCESS_KEY: test
+MHS_DYNAMODB_ENDPOINT_URL: http://localhost:8000
```

Comment:
Can you change the name to something NHAIS related?

Reply:
changed